### DISCUSSION TITLE
Query issue search with labels

### DISCUSSION THREAD

**Discussion by Teyler7** [community/community#13026] [Projects and Issues]
Hello!

I wanted to request an easier way to add issues based on a labels filter. 

For example, there may be several issues in a given repo with a label `bug` on them. I want to be able to create a bug sprint view quickly by running `repo:[REPO-NAME]` (already implemented) `label:bug` in the `Add Item` search/ input. The result should be a list of issues / Pull Requests in the given repo that also have the matching label provided. The reason behind this request is I believe it would be handy to quickly add issues with filters for organizations that have a labeling system linked to their sprints/ cycles.

I am new to discussions and this feature may already be implemented. I respect you all and thanks for allowing this discussion to exist in the first place 😃  Let me know if there is anything I can provide to be helpful.

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
